### PR TITLE
PIMS-2554, 2555 (Exemption Approval)

### PR DESCRIPTION
**Pull request by jdtoombs**
Fixed all the merge conflicts/ updated tweaks, no longer draft pr!

### CODE REVIEW

File: backend/dal/Services/Concrete/ProjectService.cs [hunk 1]
```diff
@@ -754,6 +754,23 @@ public Project SetStatus(Project project, Workflow workflow)
                         }
                     });
                     break;
+                    case ("AP-EXE"): // Approve for ERP Exemption
```
Comment (Fosol) [inline backend/dal/Services/Concrete/ProjectService.cs:757]: This looks identical to `case ("AP-ERP")`.  If it is, just allow them both to use the same code.
```
case ("AP-EXE"):
case ("AP-ERP"):
// code here
```

File: backend/dal/Helpers/Extensions/ProjectExtensions.cs [hunk 2]
```diff
@@ -41,11 +41,11 @@ public static IQueryable<Entity.Project> GenerateQuery(this PimsContext context,
                 .Include(p => p.Agency).ThenInclude(a => a.Parent)
                 .AsNoTracking();
 
-            if (filter.AccessDisposal.HasValue && filter.AccessDisposal.Value)
+            if (filter.AccessWorkflow.HasValue && filter.AccessWorkflow.Value)
```
Comment (Fosol) [inline backend/dal/Helpers/Extensions/ProjectExtensions.cs:44]: We should extract as configurable in the appsettings.json later.

File: backend/entities/Models/ProjectFilter.cs [hunk 3]
```diff
@@ -39,10 +39,10 @@ public class ProjectFilter : PageFilter
         public bool? CreatedByMe { get; set; }
 
         /// <summary>
-        /// get/set - Filter by workflow - ACCESS-DISPOSAL.
+        /// get/set - Filter by workflow - ACCESS-DISPOSAL or ACCESS-EXEMPTION.
         /// </summary>
         /// <value></value>
-        public bool? AccessDisposal { get; set; }
+        public bool? AccessWorkflow { get; set; }
```
Comment (Fosol) [inline backend/entities/Models/ProjectFilter.cs:45]: I'm just realizing we're using the work `ACCESS` instead of `ASSESS`...  Ugh

Comment (jdtoombs) [reply backend/entities/Models/ProjectFilter.cs:45]: haha i thought it was supposed to be that but then I thought I was missing something!

**Pull request merged**